### ISSUE TITLE
Pylance not checking __init__.py prior to Importing. 

### ISSUE THREAD

**Issue by Rushyanth111** [microsoft/pylance-release#298]
## Environment data
-   Language Server version: v2020.8.3
-   OS and version: Ubuntu 18.04 LTS, Kernel: 5.4.0-42-generic
-   Python version (& distribution if applicable, e.g. Anaconda): v3.7.7

## Expected behaviour

Correctly Import from Package rather than a Module.
Resulting in:
```py
from abcpackage import abc
```

## Actual behaviour

When Importing Objects from a package that has many subpackages, the auto-importing mechanism automatically chooses a module from the package, rather than reading the `__init__.py` of the package.

```py
from abcpackage.abcmodule import abc
```

## Code Snippet / Additional information

![VScode Import](https://user-images.githubusercontent.com/42034084/91634883-841c1d00-ea11-11ea-81a8-2f636c64daac.gif)

**Comment by heejaechang** [comment]
currently, in next release, we only support "from x import y as y" form, but not "from x import *" form in "auto import"

let me play with it to add this support later.

**Comment by heejaechang** [comment]
this is a dupe of https://github.com/microsoft/pylance-release/issues/291

will close this issue and track this issue on the other one.

**Comment by Rushyanth111** [comment]
I do not know if that will help? Considering most libraries follow the standard of putting any imports that are recommended in their package `__init__.py`? The Issue Linked is only asking for more of the modules to be scanned so that all of the imports show up. 

The reason I reported this as a bug is that many libraries place the "public" interface within the `__init__.py`, so really should be no reason to dig deeper when the aforementioned function is found at the package level import.

**Comment by jakebailey** [comment]
They are about the same thing; determining which import is the "correct" one offer in auto-import completions. We're closing issues as needed in order to manage what we know we need to do under the hood to make sure this works as expected.

Our next release improves this greatly, so I think you'll be interested to try it (hopefully tomorrow).

**Comment by tktk0430** [comment]
Is this issue already solved ?

**Comment by bocong** [comment]
I still seem to be having this issue, unfortunately. Can write up a repro if unexpected, but at what version was this considered fixed?

**Comment by jakebailey** [comment]
This issue was closed as a duplicate of #291. If your case differs from that issue, please file a new issue.
